src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/HotReload/ProjectHotReloadSessionManager.cs

```diff
@@ -39,7 +39,7 @@ public ProjectHotReloadSessionManager(
         Lazy<IProjectHotReloadNotificationService> projectHotReloadNotificationService,
         IProjectHotReloadLaunchProvider launchProvider,
         IProjectHotReloadBuildManager buildManager,
-        IProjectHotReloadAgent2 hotReloadAgent)
+        IProjectHotReloadAgent hotReloadAgent)
```

Comment:
Maybe we can even consider remove `ProjectHotReloadSessionManager` in the future, the existence of `ProjectHotReloadSessionManager` no longer making sense considering that the `ProjectHotReloadSession` now leverage `HotReloadLaunchProvider` and `HotReloadBuildManager` to relaunch and rebuild.

Reply:
Sounds good.